src/cal/rad_to_refl.py

```diff
@@ -0,0 +1,201 @@
+import xml.etree.ElementTree as ET
+import rasterio
+import numpy as np
+import math
+import os
+
+"""
+CURRENT ALGORITHM DOESN'T WORK WITH ATMCORR.TIF FILES
+"""
+
+# This is a large dictionary of the Earth-Sun distance in AU.
+# Goes by date. Hopefully nothing was misinputted
+date_distance = {'JAN01':0.98331,'JAN02':0.98330,'JAN03':0.98330,'JAN04':0.98330,
```

Comment:
May I suggest that this table is moved to a separate file? It can be imported in a way like:
```python
from distances import date_distance
```
It will also help with the readability of this file.